packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue

```diff
@@ -87,13 +92,19 @@ const emit = defineEmits(['update-expression', 'update-source']);
 const name = getInitialName(props.model, props.initialId);
 const unit = getInitialUnits(props.model, props.initialId);
 const description = getInitialDescription(props.model, props.initialId);
+const isExpressionEmpty = ref(false);
 
 const concept = ref('');
 const sourceOpen = ref(false);
 const showOtherConfigValueModal = ref(false);
 
 const getOtherValuesLabel = computed(() => `Other Values(${otherValueList.value?.length})`);
 
+function onExpressionChange(value) {
+	isExpressionEmpty.value = isNumberInputEmpty(value);
+	emit('update-expression', { id: props.initialId, value });
+}
```

Comment:
If the expression is empty, which I assume to be invalid ... do we still want to emit an update with the invalid value?

likewise similar logic in `onParameterChange`

Reply:
change made